### PR TITLE
release notes generation script

### PR DESCRIPTION
**Pull request by srini100**
Script to generate draft notes for review before release and final notes for publishing on Github release.

### CODE REVIEW

**Comment by srini100** [comment]
This script doesn't work with Python3 currently. Will resolve that in a separate PR.

File: tools/release/release_notes.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
+#Copyright 2018 gRPC authors.
```
Comment (lidizheng) [inline tools/release/release_notes.py]: 2019 The gRPC Authors

File: tools/release/release_notes.py [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+    if langs_pr["other"]:
+        file.write("Other\n---\n\n")
+        file.write("\n".join(langs_pr["other"]))
+        file.write("\n")
+        file.write("\n")
+
+
+def build_args_parser():
+    import argparse
+    parser = argparse.ArgumentParser()
```
Comment (lidizheng) [inline tools/release/release_notes.py]: Write a small description here might be helpful. Personally, I would like to know "rel" means "release" when I execute this script.

File: tools/release/release_notes.py (hunk 1 above)
Comment (srini100) [reply tools/release/release_notes.py]: Done

File: tools/release/release_notes.py (hunk 2 above)
Comment (srini100) [reply tools/release/release_notes.py]: Done

Comment (srini100) [reply tools/release/release_notes.py]: Changed rel_ to release_. The help string has the description.

**Comment by srini100** [comment]
Unrelated C# test flakes.

**Pull request merged**